<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin_ui</class>
 <widget class="QMainWindow" name="MainWin_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDot Dot Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainTab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Benvenuti</string>
       </attribute>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Luxi Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table border=&quot;0&quot; style=&quot;-qt-table-type: root; margin-top:18px; margin-bottom:18px; margin-left:18px; margin-right:18px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/images/logo&quot; /&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="autoFormatting">
           <set>QTextEdit::AutoAll</set>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEsegui">
    <property name="title">
     <string>&amp;Esegui</string>
    </property>
    <addaction name="actionExec"/>
    <addaction name="actionView"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Modifica</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionDrawLine"/>
    <addaction name="actionPreamble"/>
    <addaction name="separator"/>
    <addaction name="actionSetGraphvizPath"/>
   </widget>
   <widget class="QMenu" name="menuInserisci">
    <property name="title">
     <string>Elementi</string>
    </property>
    <addaction name="actAddElement"/>
    <addaction name="actAssociateElement"/>
   </widget>
   <widget class="QMenu" name="menuFinestra">
    <property name="title">
     <string>Finestra</string>
    </property>
    <addaction name="actPrevTab"/>
    <addaction name="actNextTab"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuInserisci"/>
   <addaction name="menuEsegui"/>
   <addaction name="menuFinestra"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dock1">
   <property name="maximumSize">
    <size>
     <width>239</width>
     <height>333</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Opzioni</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>190</height>
        </size>
       </property>
       <property name="title">
        <string>Opzioni</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Engine</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QComboBox" name="cmbEngine">
          <item>
           <property name="text">
            <string>dot</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>neato</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>fdp</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>circo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>twopi</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Formato File</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QComboBox" name="cmbDestType">
          <item>
           <property name="text">
            <string>pdf</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>jpg</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>png</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>gif</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Largezza</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Altezza</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QSpinBox" name="edtWidth">
          <property name="value">
           <number>7</number>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="edtHeight">
          <property name="value">
           <number>8</number>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>102</height>
        </size>
       </property>
       <property name="title">
        <string>Compilazione / Visualizza</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnCompile">
          <property name="text">
           <string>Compila [F2]</string>
          </property>
          <property name="icon">
           <iconset resource="../resource.qrc">
            <normaloff>:/images/compile.png</normaloff>:/images/compile.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="LedFrame" name="ledFrame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="btnView">
          <property name="text">
           <string>Visualizza [F3]</string>
          </property>
          <property name="icon">
           <iconset resource="../resource.qrc">
            <normaloff>:/images/images/view.png</normaloff>:/images/images/view.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dock2">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Elementi</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Elementi</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QListWidget" name="elementsList">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="separator"/>
   <addaction name="actPrevTab"/>
   <addaction name="actNextTab"/>
   <addaction name="separator"/>
   <addaction name="actionPreamble"/>
   <addaction name="actAddElement"/>
   <addaction name="actAssociateElement"/>
   <addaction name="separator"/>
   <addaction name="actionExec"/>
   <addaction name="actionView"/>
   <addaction name="separator"/>
   <addaction name="actionSetGraphvizPath"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Esci</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Nuovo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Apri</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Salva</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/images/saveas.png</normaloff>:/images/images/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Sa&amp;lva con nome</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Chiudi</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionExec">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/compile.png</normaloff>:/images/compile.png</iconset>
   </property>
   <property name="text">
    <string>Compila</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionView">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/images/view.png</normaloff>:/images/images/view.png</iconset>
   </property>
   <property name="text">
    <string>Visualizza</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/images/zoomIn.png</normaloff>:/images/images/zoomIn.png</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/images/zoomOut.png</normaloff>:/images/images/zoomOut.png</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDrawLine">
   <property name="text">
    <string>Disegna linea</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionPreamble">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/preamble.png</normaloff>:/images/preamble.png</iconset>
   </property>
   <property name="text">
    <string>Preambolo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actAddElement">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/images/add.png</normaloff>:/images/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Aggiungi</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actAssociateElement">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/images/associate.png</normaloff>:/images/images/associate.png</iconset>
   </property>
   <property name="text">
    <string>Associa</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actPrevTab">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/images/prev.png</normaloff>:/images/images/prev.png</iconset>
   </property>
   <property name="text">
    <string>TabIndietro</string>
   </property>
   <property name="shortcut">
    <string>Meta+Left</string>
   </property>
  </action>
  <action name="actNextTab">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/images/next.png</normaloff>:/images/images/next.png</iconset>
   </property>
   <property name="text">
    <string>TabAvanti</string>
   </property>
   <property name="shortcut">
    <string>Meta+Right</string>
   </property>
  </action>
  <action name="actionSetGraphvizPath">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/images/configure.png</normaloff>:/images/images/configure.png</iconset>
   </property>
   <property name="text">
    <string>setGraphvizPath</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LedFrame</class>
   <extends>QFrame</extends>
   <header location="global">LedFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbEngine</tabstop>
  <tabstop>cmbDestType</tabstop>
  <tabstop>edtWidth</tabstop>
  <tabstop>edtHeight</tabstop>
  <tabstop>btnCompile</tabstop>
  <tabstop>btnView</tabstop>
  <tabstop>mainTab</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
